dropwizard-example/src/test/java/com/example/helloworld/resources/PersonResourceTest.java

```diff
@@ -14,10 +13,7 @@
 import java.util.Optional;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.reset;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Are we fine with wildcard imports ? This will bring in all the army and unnecessarily increase source code. Just wanted to know for self-reference as well ?

Reply:
Good call, let’s expand this out please